packages/fluent-theme/src/components/preChatActivity/StarterPromptsCardAction.module.css

```
@@ -44,18 +44,24 @@
     outline-offset: -2px;
   }
 
-  @media (prefers-reduced-motion: no-preference) {
+  @media (prefers-reduced-motion: reduce) {
     &:empty::after,
     &:empty::before {
-      animation-duration: 3s;
-      animation-iteration-count: infinite;
-      animation-name: blueprintAnimation;
-      animation-timing-function: linear;
+      animation-delay: -1s !important;
+      animation-play-state: paused !important;
```

```suggestion
      animation-delay: -1s;
      animation-play-state: paused;
```
Why raising importance like this?

---

I'm ok to have it in tests, but the rule for CSS should be avoid `!important` at any costs

---

Because I was just copying from yours. 🤣